sdk/attestation/azure-security-attestation/azure/security/attestation/_administration_client.py

```diff
@@ -4,316 +4,736 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------
 
-from typing import List, Any, Optional, TYPE_CHECKING
-
-from azure.core import PipelineClient
-from six import python_2_unicode_compatible
+from typing import Dict, List, Any, TYPE_CHECKING, Tuple, Union
+import base64
+from threading import Lock
 
-if TYPE_CHECKING:
-    # pylint: disable=unused-import,ungrouped-imports
-    from typing import Any
+from cryptography.hazmat.primitives import serialization
+from cryptography.x509 import load_pem_x509_certificate
 
-    from azure.core.credentials import TokenCredential
-    from azure.core.pipeline.transport import HttpRequest, HttpResponse
+from azure.core.tracing.decorator import distributed_trace
 
 from ._generated import AzureAttestationRestClient
 from ._generated.models import (
-    AttestationType, 
-    PolicyResult as GeneratedPolicyResult, 
-    PolicyCertificatesResult, 
-    JSONWebKey, 
-    AttestationCertificateManagementBody, 
+    AttestationType,
+    PolicyResult as GeneratedPolicyResult,
+    PolicyCertificatesResult as GeneratedPolicyCertificatesResult,
+    JSONWebKey,
+    AttestationCertificateManagementBody,
     StoredAttestationPolicy as GeneratedStoredAttestationPolicy,
-    PolicyCertificatesModificationResult as GeneratedPolicyCertificatesModificationResult
+    PolicyCertificatesModificationResult as GeneratedPolicyCertificatesModificationResult,
 )
 from ._configuration import AttestationClientConfiguration
 from ._models import (
-    AttestationSigner, 
-    AttestationToken, 
-    AttestationResponse, 
-    AttestationSigningKey, 
-    PolicyCertificatesModificationResult,
-    PolicyResult,
-    AttestationTokenValidationException
+    AttestationSigner,
+    AttestationToken,
+    AttestationPolicyResult,
+    AttestationPolicyCertificateResult,
 )
-import base64
-from azure.core.tracing.decorator import distributed_trace
-from threading import Lock, Thread
+from ._common import pem_from_base64, validate_signing_keys, merge_validation_args
+
+if TYPE_CHECKING:
+    # pylint: disable=unused-import,ungrouped-imports
+    from azure.core.credentials import TokenCredential
 
 
 class AttestationAdministrationClient(object):
+    # pylint: disable = line-too-long
```

Comment:
If you add the `protected-access` suppression here, you can do away with the rest of the suppression comments throughout the class. I think it would be cleaner and the expanded scope would be justified because of how often generated code needs to be accessed, but it's ultimately up to you
```suggestion
    # pylint: disable = line-too-long, protected-access
```